definitions/npm/jest_v14.x.x/flow_>=v0.23.x/jest_v14.x.x.js

```diff
@@ -65,8 +97,16 @@ declare var jest: {
   setMock(moduleName: string, moduleExports: any): void;
   unmock(moduleName: string): void;
   useFakeTimers(): void;
+  useRealTimers(): void;
 }
 
 declare var jasmine: {
   DEFAULT_TIMEOUT_INTERVAL: number;
+  any(val: mixed): void;
```

Comment:
I've added a `JestAsymmetricEqualityType` type, omitting the `jasmineToString` method, since it's not documented on http://jasmine.github.io/2.4/introduction.html